TicTacToeRunner/Runner.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using TicTacToeApp;
+using TicTacToeUserInterface;
+using System.Collections.Generic;
+
+namespace TicTacToeRunner
+{
+    class Runner
+    {
+        public TicTacToe newGame;
+        public UserInterface gameUI;
+        public bool isGameOver = false;
+
+
+        static void Main(string[] args)
+        {
+            var runner = new Runner();
+            runner.createGame();
+        }
+
+
+        public void createGame()
+        {
+            gameUI = new UserInterface();
+            if (gameUI.startNewGame())
+            {
+                var options = new Options(gameUI.setMarkers());
+                newGame = new TicTacToe(options.P1_MARKER, options.P2_MARKER);
+
+                while (!isGameOver)
+                {
+                    playGameLoop();
+                }
+            }
+        }
+
+
+        public Dictionary<int, string> createPrimitiveBoard()

```

Comment:
To me this seems like a function that the board would be responsible for. I was thinking that the board could provide a representation of itself as a string, which the runner would pass to the UI for printing.
Looking at the UI here, it's named `gameUI` and the method called to print the board is `displayBoard`...
That's a pretty good indication that we should actually pass in the whole board object since the method seems to have knowledge of the object.
Maybe there could be an entity that is solely responsible for sending and receiving messages to/fro the user  (printing to the console). Like an abstract class called `UI`. That class would likely only need to consume strings through something like `read` and `write` methods. 
If there were then a `GameUI` class that extended the `UI` class and used those print methods inside functions that asked the board for its string representation and then sent that representation to its inherited print method, it would be more clear where displayables specific to tic tac toe are handled vs where user interaction is handled.

👍